genai/api/npc_chat_api/Dockerfile

```diff
@@ -25,9 +25,6 @@ RUN pip install --no-cache-dir -r requirements.txt
 
 COPY src /app/
 
-# TODO: Use ConfigMap instead
-COPY config /app/config
```

Comment:
Should the two files in https://github.com/googleforgames/GenAI-quickstart/tree/main/genai/api/npc_chat_api/config be removed as part of this change as well (since the contents is now in the configmap)?

Reply:
Do we have any preference between deleting the .toml files and having the configmaps directly in the k8s.yaml (as it is now), or leaving in the .toml files and using Kustomize to generate the configmaps?

Reply:
I don't have a strong preference as long as the data has a single source of truth.